code/game/machinery/limbgrower.dm

```diff
@@ -31,6 +31,7 @@
 	stored_research = new /datum/techweb/specialized/autounlocking/limbgrower
 	. = ..()
 	AddComponent(/datum/component/plumbing/simple_demand)
+	interaction_flags_machine |= INTERACT_MACHINE_OFFLINE
```

Comment:
So you're using `INTERACT_MACHINE_OFFLINE` so that the limbgrower can be programmed while the power is off, how does that work specifically with this UI? I thought the limbgrower instantly printed the limb from lathe datums.

Reply:
You are right. I enabled used INTERACT_MACHINE_OFFLINE for the limbgrower since I was trying to standardize it for plumbing machinery, but in this case its better to not have it.
Problem is that the limb grower can still grow limbs instantly, even though power is off.
Going to remove this line